networkx/algorithms/lowest_common_ancestors.py

```diff
@@ -130,6 +152,21 @@ def lowest_common_ancestor(G, node1, node2, default=None):
     The lowest common ancestor of node1 and node2,
     or default if they have no common ancestors.
 
+    Examples
+    --------
+    >>> G = nx.DiGraph()
+    >>> G.add_edges_from([(0, 1), (0, 2), (2, 3), (2, 4), (1, 6), (4, 5)])
+    >>> nx.lowest_common_ancestor(G, 3, 5)
+    2
+
+    We can also set `default` argument as below. The value of default is returned
+    if there are no common ancestors of given two nodes.
+
+    >>> G = nx.DiGraph()
+    >>> G.add_edges_from([(4, 5), (12, 13)])
+    >>> print(nx.lowest_common_ancestor(G, 12, 5, default="No common ancestors!"))
+    No common ancestors!
```

Comment:
This is definitely a nit, but it feels strange to have this wrapped in `print()` as that would really only make sense if you were expecting the output to be a string. This is of course the case for this contrived example, but I think it'd be ever so slightly more clear to remove the print:

```suggestion
    >>> nx.lowest_common_ancestor(G, 12, 5, default="No common ancestors!")
    'No common ancestors!'
```